content/articles/dkim-record.markdown

```diff
@@ -0,0 +1,62 @@
+---
+title: DKIM records
+excerpt: This article explains how DKIM records work.
+categories:
+- DNS
+- Email
+---
+
+# DKIM Records
+
+* TOC
+{:toc}
+
+---
+
+DKIM stands for DomainKeys Identified Email and provides a way to validate that an organization delivering an email has the right to do so.
+
+## Setting up DKIM
+
+DKIM requires the addition of public keys into your DNS zone. The key is often provided to you by the organization that is sending your email, for example [SendGrid](https://sendgrid.com/docs/Glossary/dkim.html), [Postmark](http://support.postmarkapp.com/customer/portal/articles/64739-what-is-dkim-), or [Google Apps](https://support.google.com/a/answer/174124?hl=en). The key will either be inserted directly into your zone as a TXT record, or it will be a CNAME pointing to the key in your provider's DNS.
+
+If you are given a string representing the DKIM, it will usually look something like this:
+
+    k=rsa; t=s; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDGMjj8MVaESl30KSPYdLaEreSYzvOVh15u9YKAmTLgk1ecr4BCRq3Vkg3Xa2QrEQWbIvQj9FNqBYOr3XIczzU8gkK5Kh42P4C3DgNiBvlNNk2BlA5ITN/EvVAn/ImjoGq5IrcO+hAj2iSAozYTEpJAKe0NTrj49CIkj5JI6ibyJwIDAQAB
+
+This should be inserted into a TXT record. You can do this by following the instructions for [creating a record](http://support.dnsimple.com/articles/advanced-editor/#create-a-record), selecting TXT as the record type, and entering the string you were given in the Content field.
+
+Your provider will also have given you a specific subdomain to use, usually something like this:
+
+   something.\_domainkey
+
+You will enter this subdomain in the "Name" field.
+
+Note that if your provider gives you a fully-qualified name that ends with your domain name you should NOT include your domain name in the "Name" field when you add the TXT record. For example, if you are given `pm.\_domainkey.yourdomain.com` then you will only enter `pm.\_domainkey` in the "Name" field.
```

Comment:
Thanks. I was trying to track down a missing backslash - good to know they are not needed in single quotes.